src/app.js

```diff
@@ -0,0 +1,71 @@
+require("babel-runtime/regenerator");
+require("webpack-hot-middleware/client?reload=true");
+require('./favicon.ico');
+
+require("./scss/cart.scss");
+require("./scss/products.scss");
+require("./scss/login.scss");
+require("./scss/carousel.scss");
+require("./scss/query.scss");
+require("./scss/home.scss");
+require("./scss/common.scss");
+
+
+const header = `<div class="row">
+<nav>
```

Comment:
Should be header tag. Nav should only cover navigation and not the logo/cart

Reply:
Done